### ISSUE TITLE
Tooltip displayed on click when selected trigger is hover

### ISSUE THREAD

**Issue by guoliang** [infor-design/enterprise#2185]
**Describe the bug**
I have a button which has a tooltip on it, and when I click on it it will switch to another template. But due to the on button click trigger the tooltip will be displayed on a corner in the new template 

**To Reproduce**
1. Go to http://htmlpreview.github.io/?https://github.com/guoliang/soho-sandbox/blob/master/index.html
2. Click on button to see the bug

**Expected behavior**
If the selected trigger is "hover" it should not add extra that tooltip will be displayed on click.

**Version**
- ids-enterprise v4.18

**Platform**
 - Chome

**Additional context**
Refer to this [line](https://github.com/infor-design/enterprise/blob/bb27ba44d631d664f01673d19f78cd9a0e1a8d54/src/components/tooltip/tooltip.js#L260)

**Comment by tmcconechy** [comment]
@guoliang Sorry im not following this issue. Maybe try to put it in the template format so the steps are clear? Im not certain what your asking for here.

**Comment by guoliang** [comment]
Here's an example
http://htmlpreview.github.io/?https://github.com/guoliang/soho-sandbox/blob/master/index.html

**Comment by tmcconechy** [comment]
Not 100% sure how to approach this. Maybe you can destroy the tooltip when you switch view?

**Comment by guoliang** [comment]
I already tried it, and it does destroy the object. However that line which I linked to is triggered due to button click, and the tooltip is displayed again. I could resolve the issue by just removing the `.on("click...` part. I'm not sure if it will cause issue else where though, but instinctively I think if I have picked `settings.trigger: hover` then the tooltip should display on however only and not on click as well.
